Mathlib/CategoryTheory/Limits/IsLimit.lean

```diff
@@ -568,7 +568,7 @@ structure IsColimit (t : Cocone F) where
 
 attribute [reassoc (attr := simp)] IsColimit.fac
 
--- Porting note: simp can prove this. Linter claims it still is tagged with simp
+-- Porting note (#10618): simp can prove this. Linter claims it still is tagged with simp
```

Comment:
What is going on with this one? Is this a bug in the simpNF linter?

Reply:
This question should probably be reported in the corresponding issue #10618. 

I'm going to copy-paste it there.

Reply:
Now this question can be found [here](https://github.com/leanprover-community/mathlib4/issues/10618#issuecomment-1955018632).